crates/esthri/src/ops/sync.rs

```diff
@@ -177,6 +178,9 @@ fn create_dirent_stream<'a>(
                 yield Err(entry.err().unwrap().into());
                 return;
             };
+            if entry.file_name().to_string_lossy().contains(TEMP_FILE_PREFIX) {
```

Comment:
A somewhat unrelated fix, but have included it here just so we don't have branches on top of branches on top of branches. A little more info in the commit message: https://github.com/swift-nav/esthri/pull/255/commits/9cdafb7629c30edf547f2ce223b21258793b170c